Source/ACE.Database/SerializedShardDatabase.cs

```diff
@@ -237,6 +237,21 @@ public void GetWieldedItems(uint parentId, Action<List<Biota>> callback)
 
         }
 
+        public Biota GetObjectByGuid(uint guid)
```

Comment:
Why the addition of GetObjectByGuid. This seems to do the same thing as GetBiota, except it returns a biota that doesn't use our context caching system.